cli/src/factory_ops/boxing/init

```diff
@@ -95,6 +95,17 @@ fi
 # Prepare shutdown
 ensure_healthy_cluster
 
+echo "INFO: Resolving service user credentials" | tee -a ${LOG_FILE}
+# FIXME these values might be encrypted
+service_user_name=$(salt-call pillar.get system:service_user:name --output=newline_values_only)
+service_user_secret=$(salt-call pillar.get system:service_user:password --output=newline_values_only)
+
+if [[ -z "$service_user_name" || -z "$service_user_secret" ]]; then
+    msg="ERROR: credentials for a service user are not found"
+    echo "$msg" | tee -a ${LOG_FILE}
+    die "$msg"
```

Comment:
I don't see definition of "die"

Reply:
how about that one ? https://github.com/Seagate/cortx-prvsnr/blob/e6d15babe156505f810b912ccd0918112cc42788/cli/src/factory_ops/boxing/init#L54